test/functional/plugin/shada_spec.lua

```diff
@@ -715,6 +722,7 @@ describe('In autoload/shada.vim', function()
         '  # Unexpected enum value: expected one of 0 (CHARACTERWISE), '
         .. '1 (LINEWISE), 2 (BLOCKWISE)',
         '  + rt   type         10',
+        '  + ru   is_unnamed   FALSE',
```

Comment:
Note what tests are testing. Just adding this line to the failing tests is not enough.